voorhoede/head-start
# feature: Page Partial Block

# Changes

- Adds Page Partial Block. It renders one or more reusable page partials, each containing a set of content blocks. See its README.md for more info.
- Adds migration for the Page Partial Block. This migration contains references to the new Text Image Block, so #84 should be merged first.
- Registers the Block in Blocks.astro|d.ts and uses its fragment in the home page and generic page queries.

# Associated issue

Resolves #34 

# How to test

1. Open preview link
2. Navigate to `/nl/`
3. Verify a Text Block is rendered from within a Page Partial Block.
4. Create your own Page Partial Block in the CMS (in the `nested-pages` environment)
5. Verify it renders on the page(s) you added it to.

# Checklist

- ~~I have added/updated tests to prove that my feature works (if not possible please explain why)~~
- ~~I have added a decision log entry if the change affects the architecture or changes a significant technology~~
